lib/generators/install/templates/initializer.rb.erb

```diff
@@ -4,6 +4,30 @@ VertexClient.configure do |config|
   config.trusted_id = ENV.fetch('VERTEX_TRUSTED_ID')
   config.soap_api = ENV.fetch('VERTEX_SOAP_API')
 
+  # Global timeout options passed to Savon
+  config.read_timeout = 10
+  config.open_timeout = 10
+
+  # Per-resource timeout options
+  config.resources = {
```

Comment:
I was a little thrown off by this being called resources. What do you think about `resource_configs`, which would match the code that references `resource_config` (singular)?

Reply:
Opted for a sigular variant :+1: